mempool/src/shared_mempool/coordinator.rs

```diff
@@ -191,7 +190,7 @@ async fn handle_event<V>(
         }
         Event::LostPeer(metadata) => {
             counters::SHARED_MEMPOOL_EVENTS
-                .with_label_values(&["lost_peer".to_string().deref()])
+                .with_label_values(&["lost_peer"])
```

Comment:
interesting

Reply:
Yeah, I don't really know why someone went through these lengths to convert an `&str` back into an `&str`